code/game/objects/items/bodybag.dm

```diff
@@ -84,6 +84,33 @@
 	else
 		icon_state = icon_opened
 
+/obj/structure/closet/body_bag/take_contents()
+	..()
+	var/mob/living/L = usr
```

Comment:
pass an arg instead of using usr when at all possible (Topics are the usual exception)